docs/docs/modules/data_connection/document_loaders/pdf.mdx

```diff
@@ -129,6 +129,11 @@ data = loader.load()
 
 ## Using Unstructured
 
+The `unstructured[all-docs]` package currently supports loading of text files, powerpoints, html, pdfs, images, and more.
+
+```bash
+pip install unstructured[all-docs]
```

Comment:
is `unstructured[pdf]` sufficient here?

Reply:
yes that would be sufficient and more appropriate